src/server/Document/component.jsx

```diff
@@ -65,8 +65,16 @@ const Document = ({
       </head>
       <body>
         {/* disabling the rule that bans the use of dangerouslySetInnerHTML until a more appropriate implementation can be implemented */}
-        {/* eslint-disable-next-line react/no-danger */}
-        <div id="root" dangerouslySetInnerHTML={{ __html: app }} />
+        <div
+          id="root"
+          // eslint-disable-next-line react/no-danger
+          dangerouslySetInnerHTML={{ __html: app }}
+          style={{
+            minHeight: '100vh',
+            display: 'flex',
+            flexDirection: 'column',
+          }}
```

Comment:
I think rather than using the `style` attribute, I think you may want to use import `syledComponents` and use a `styled.div`:

```
styled.div`minHeight: '100vh', display: 'flex', flexDirection: 'column'`;
```

That way you are avoiding the use of inline styles as is convention in the codebase